railties/lib/rails/application/configuration.rb

```diff
@@ -214,6 +214,10 @@ def load_defaults(target_version)
             active_support.hash_digest_class = OpenSSL::Digest::SHA256
             active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
           end
+
+          if respond_to?(:action_controller)
+            action_controller.raise_on_open_redirects = true
```

Comment:
Needs to update the configuring.md guide with this new config and its default and base value.

Reply:
I think this is done. Take another look when you have time.

Reply:
@gmcgibbon I think the config needs to be added here as well:
https://github.com/rails/rails/blob/4d3a8a0bf0bc63cc0adc59d4f661fe0cce9f3ec2/guides/source/configuring.md#for-70-defaults-from-previous-versions-below-and